### PR TITLE
Add a sample for quick start guide

### PR DESCRIPTION
**Pull request by nilasini**
## Purpose
Openid connect sso demonstration using two applications Dispatch and Swift.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: QSGDemo/Dispatch/src/main/org/wso2/sample/identity/oauth2/OAuth2ClientServlet.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+// This is the servlet which handles OAuth callbacks.
```
Comment (denuwanthi) [inline QSGDemo/Dispatch/src/main/org/wso2/sample/identity/oauth2/OAuth2ClientServlet.java]: its more clear to use /** as a class level comment.

File: QSGDemo/Dispatch/src/main/webapp/oauth2-authorize-user.jsp [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                .setScope(scope);
+
+
+//build the new response mode with form post
+        OAuthClientRequest authzRequest = oAuthAuthenticationRequestBuilder.buildQueryMessage();
+        //response.ge
+        response.sendRedirect(authzRequest.getLocationUri());
+        return;
+
+} catch (Exception e) {
```
Comment (ruwanta) [inline QSGDemo/Dispatch/src/main/webapp/oauth2-authorize-user.jsp]: Need to catch the specific exceptions.

File: QSGDemo/Dispatch/src/main/webapp/oauth2-authorize-user.jsp [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        oAuthAuthenticationRequestBuilder
+                .setClientId(consumerKey)
+                .setRedirectURI((String) session.getAttribute(OAuth2Constants.CALL_BACK_URL))
+                .setResponseType(authzGrantType)
+                .setScope(scope);
+
+
+//build the new response mode with form post
+        OAuthClientRequest authzRequest = oAuthAuthenticationRequestBuilder.buildQueryMessage();
+        //response.ge
```
Comment (ruwanta) [inline QSGDemo/Dispatch/src/main/webapp/oauth2-authorize-user.jsp]: ?? response.ge

File: QSGDemo/Dispatch/src/main/webapp/oauth2-authorize-user.jsp [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    window.location = "index.jsp";
+</script>
+
+<%
+    }
+%>
+
+
+
+
```
Comment (ruwanta) [inline QSGDemo/Dispatch/src/main/webapp/oauth2-authorize-user.jsp]: EOF

File: QSGDemo/Swift/src/main/org/wso2/sample/identity/oauth2/SampleContextEventListener.java [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        properties = new Properties();
+        try {
+            properties.load(servletContextEvent.getServletContext().
+                    getResourceAsStream("/WEB-INF/classes/swift.properties"));
+        } catch (IOException e) {
+            LOGGER.log(Level.SEVERE, e.getMessage(), e);
+        }
+    }
+
+    public void contextDestroyed(ServletContextEvent servletContextEvent) {
```
Comment (ruwanta) [inline QSGDemo/Swift/src/main/org/wso2/sample/identity/oauth2/SampleContextEventListener.java]: Please put a comment why this is empty

File: QSGDemo/Swift/src/main/resources/swift.properties [hunk 6]
```diff
@@ -0,0 +1,19 @@
+consumerKey=ZJw0BrpmkFriZl1FK7LdPSRLYvoa
+
```
Comment (ruwanta) [inline QSGDemo/Swift/src/main/resources/swift.properties]: Why we have empty line between properties. It looks odd.

File: QSGDemo/Dispatch/src/main/webapp/style.css [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+	color:#666;
+	text-decoration:none;
+	display: inline-block;
+	padding:5px 10px;
+	background:#e5e5e5;
+	line-height:100%;
+	-moz-border-radius:3px;
+	-webkit-border-radius:3px;
+	border-radius:3px;
+	}
```
Comment (mefarazath) [inline QSGDemo/Dispatch/src/main/webapp/style.css]: EOF

**Pull request merged**